stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -137,29 +143,82 @@ There is no default size to preserve what's being ultimately rendered.
 
 ## Colors
 
-The different colors available are: `text` (default), `highlighted-text`, `info`, `success`, `warning`, `error` which
-all refer to OKP4 palette and theming.
+There are different colors available referring to OKP4 palette and themes.
+
+The basic colors are :
+
+- `text` - the default text color,
+- `highlighted-text`,
+- `info`,
+- `success`,
+- `warning`,
+- `error`
+
+The other colors are :
```

Comment:
I think you should fo further in the description, to explain the the main aim of the separation between base and complex colors.
Something like:
```suggestion
To allow the management of higher contrasts or cases where the color must remain the same according to the theme, the OKP4 palette addresses these issues by adding 3 more complex color effects:
```

Reply:
completely agree